pom.xml

```diff
@@ -10,19 +12,53 @@
   <artifactId>instance-identity</artifactId>
   <version>2.1-SNAPSHOT</version>
   <packaging>jenkins-module</packaging>
+
   <name>Instance Identity</name>
   <description>Maintains an RSA key pair that can serve as a foundation of authentication when communicating with Jenkins</description>
+  <licenses>
+    <license>
+      <name>MIT License</name>
+      <url>http://www.opensource.org/licenses/mit-license.php</url>
+    </license>
+  </licenses>
 
   <scm>
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-module.git</connection>
     <developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}-module.git</developerConnection>
     <tag>HEAD</tag>
   </scm>
 
-  <licenses>
-    <license>
-      <name>MIT License</name>
-      <url>http://www.opensource.org/licenses/mit-license.php</url>
-    </license>
-  </licenses>
+  <properties>
+    <!-- TODO remove once Jenkins core 2.16 released -->
```

Comment:
Why? if this is removed then the parent matching version will be used (which is `1.625.3` currently), so how is it going to pick anything from 2.16?